apps/server/src/modules/tool/controller/tool.controller.ts

```diff
@@ -27,4 +34,15 @@
 		const mapped: Lti11LaunchResponse = this.lti11ResponseMapper.mapAuthorizationToResponse(authorization);
 		return mapped;
 	}
+
+	@Post('externalTool')
+	async createExternalTool(
+		@Body() externalToolParams: ExternalToolParams,
+		@CurrentUser() currentUser: ICurrentUser
+	): Promise<ExternalToolResponse> {
+		// const externalTool: ExternalTool = await this.externalToolUc.createExternalTool(externalToolParams);
```

Comment:
please remove the commented code. Also, is the uc still needed if it is not used?